### PR TITLE
codegen removal

### PR DESCRIPTION
**Pull request by kentrachmat**
Thank you for your reply, in this PR i have deleted the whole codegen file

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -13,15 +13,14 @@
   <artifactId>gson-parent</artifactId>
   <version>2.9.1-SNAPSHOT</version>
   <packaging>pom</packaging>
-
+
```
Comment (eamonnmcmanus) [inline pom.xml:16]: This doesn't seem necessary?

**Review by eamonnmcmanus — COMMENTED** [review]
Thanks! We do need a signed CLA before we can merge, though.

File: pom.xml (hunk 1 above)
Comment (kentrachmat) [reply pom.xml:16]: Yes sorry, i was trying to solve the CLA problem, this is actually my first ever PR and i thought by changing my email in the next commit would solve this problem

**Comment by kentrachmat** [comment]
Hai I have signed the CLA but i still can't merge, can you help ? thank you in advance

**Pull request merged**